src-java/flowmonitoring-topology/flowmonitoring-messaging/src/main/java/org/openkilda/messaging/info/flow/UpdateFlowInfo.java

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2021 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.info.flow;
+
+import org.openkilda.messaging.info.InfoData;
+import org.openkilda.messaging.model.FlowPathDto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.Value;
+
+/**
+ * Represents update flow info.
+ */
+@Value
+@JsonSerialize
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class UpdateFlowInfo extends InfoData {
+    private static final long serialVersionUID = 1L;
+
+    @JsonProperty("flow_id")
+    protected String flowId;
+
+    @JsonProperty("flow_path")
+    protected FlowPathDto flowPath;
+
+    @JsonProperty("max_latency")
+    protected Long maxLatency;
```

Comment:
why is it Object not a primitive?

Reply:
What about the `max_latency_tier2` field?

Reply:
Because we can have a flows with null maxLatency.
Added maxLatencyTier2.